### ISSUE TITLE
README.md references non-existent example.config.json file

### ISSUE THREAD

**Issue by celowsky** [jonah-dev/lfd2-bot#43]
The **Step 3 Details** section of README.md instructs developers to copy a file named example.config.json, substituting in their own Discord API token. However, this file does not exist in this repository and cannot be found in this repository's history via `git log --all --full-history -- "**/config.*"`, which can be used to search for deleted files in VCS [according to this Stackoverflow answer](https://stackoverflow.com/questions/7203515/git-how-to-find-a-deleted-file-in-the-project-commit-history). This section of README.md should be updated to include updated instructions for config setup.

Edit 1/13/2021: Fixed a link

**Comment by JoeDailey** [comment]
@jonah-dev was working on this. At some point we switched to environment variables. (I'm recalling something about Dan and Docker) Anyways, I've doubled down on them. Note that in my dev build I don't actually write this in my shell config but I run `DISCORD_TOKEN="asdf" python bot.py` from the terminal. I do this because I'm usually switching bots and projects often. 

There are only two.
 - `DISCORD_TOKEN` is the value provided in Discord's developer portal
 - `DISCORD_ADMINS` is a string with space-separated Discord IDs

These are also important in our system unit.
```
Environment=DISCORD_TOKEN="asdf"
Environment=DISCORD_ADMINS="1234 5678"
WorkingDirectory=/home/jd/lfd2-bot
ExecStart=/home/jd/lfd2-bot/env/bin/python /home/jd/lfd2-bot/bot.py
```
